ui/partials/perf/runinfo.html

```diff
@@ -0,0 +1,3 @@
+<span ng-if="runs.length > 0" class="run-detail" tooltip="Runs: < {{runs.join(', ')}} >">
```

Comment:
Just subjective, but I wonder if the `<` and `>` might look less like htm tags injected into the tooltip, if they were `( )`?